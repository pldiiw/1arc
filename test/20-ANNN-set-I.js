const engine = new Map([['I', 0]]);
const setI = require('../src/instruction-set.js').setI;

test('set reg I to 3020', () => {
  expect(setI(engine, 3020).get('I')).toBe(3020);
});
