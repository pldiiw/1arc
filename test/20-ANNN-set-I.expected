0
3020
20
